README.Rmd

```diff
@@ -15,58 +15,46 @@ knitr::opts_chunk$set(
-[![R build status](https://github.com/jameel-institute/ji.rpkg.template/workflows/R-CMD-check/badge.svg)](https://github.com/jameel-institute/ji.rpkg.template/actions/workflows/R-CMD-check.yaml)
-[![Codecov test coverage](https://codecov.io/gh/jameel-institute/ji.rpkg.template/branch/main/graph/badge.svg)](https://app.codecov.io/gh/jameel-institute/ji.rpkg.template?branch=main)
-[![CRAN status](https://www.r-pkg.org/badges/version/ji.rpkg.template)](https://CRAN.R-project.org/package=ji.rpkg.template)
+[![Project Status: Concept - Minimal or no implementation has been done yet, or the repository is only intended to be a limited example, demo, or proof-of-concept.](https://www.repostatus.org/badges/latest/concept.svg)](https://www.repostatus.org/#concept)
+[![R build status](https://github.com/vimc/vimclimate/workflows/R-CMD-check/badge.svg)](https://github.com/vimc/vimclimate/actions/workflows/R-CMD-check.yaml)
+[![Codecov test coverage](https://codecov.io/gh/vimc/vimclimate/branch/main/graph/badge.svg)](https://app.codecov.io/gh/vimc/vimclimate?branch=main)
+[![CRAN status](https://www.r-pkg.org/badges/version/vimclimate)](https://CRAN.R-project.org/package=vimclimate)
 <!-- badges: end -->
 
-_ji.rpkg.template_ is a template package and repository on which future Jameel Institute packages are based.
-
-To use this template, select _ji.rpkg.template_ from the drop-down menu under **Repository template** when creating a new repository in the Jameel Institute organisation.
-Replace all instances of "ji.rpkg.template" with your package name. Make sure to also:
-
-1. Edit the `DESCRIPTION` as appropriate with the correct package information;
-
-2. Edit the files in `R/`, `tests/`, and `vignettes/` to suit your package;
-
-3. Remove these instructions from `README.Rmd`, and re-render the `.md` file using `devtools::render_readme()`.
+_vimclimate_ is an R package that helps access pre-made climatic variable time-series data for use in modelling.
 
 ## Installation
 
-**NOTE:** Remove or comment out installation sources as appropriate.
-
-You can install the development version of ji.rpkg.template from the Jameel Institute R-universe with:
+You can install the development version of _vimclimate_ from GitHub [GitHub](https://github.com/) using the _pak_ package:
 
-```r
-# installation from R-universe
-# install.packages(
-#   "ji.rpkg.template", 
-#   repos = c(
-#     "https://jameel-institute.r-universe.dev", "https://cloud.r-project.org"
-#   )
-# )
+``` r
+install.packages("pak")
+pak::pak("vimc/vimclimate")
 ```
 
-or from GitHub [GitHub](https://github.com/) with:
+<!-- the [VIMC R-universe](https://vimc.r-universe.dev/) with:
 
-``` r
-# install.packages("pak")
-# pak::pak("jameel-institute/ji.rpkg.template")
+```r
+installation from R-universe
+install.packages(
+  "vimclimate", 
```

Comment:
https://github.com/vimc/vimc.r-universe.dev - can you make a PR here to add this, I think it can be added immediately